### PR TITLE
Enhancing enterprise permissions page

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Enhancing features and descriptions on the Enterprise Permissions page:
Adding new Role
Adding table with roles & permissions
Adding clarifying language to the permissions descriptions where neccessary. 

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

**Comment by nm0s** [comment]
Following up on my slack comment about maybe breaking the table into two tables (account-level permissions, and project-level permissions) to avoid the scroll. Here are the account level permissions we have - pulled from the account admin permission set:
```
    ACCOUNT_LEVEL_PERMISSIONS = [
        AvailablePermissions.account_settings_read,
        AvailablePermissions.account_settings_write,
        AvailablePermissions.billing_read,
        AvailablePermissions.billing_write,
        AvailablePermissions.projects_create,
        AvailablePermissions.auth_provider_read,
        AvailablePermissions.auth_provider_write,
        AvailablePermissions.ip_restrictions_read,
        AvailablePermissions.ip_restrictions_write,
        AvailablePermissions.invitations_send,
        AvailablePermissions.invitations_modify,
        AvailablePermissions.invitations_read,
        AvailablePermissions.members_read,
        AvailablePermissions.members_write,
        AvailablePermissions.service_tokens_read,
        AvailablePermissions.service_tokens_write,
        AvailablePermissions.audit_log_read,
        AvailablePermissions.internal_webhooks_read,
        AvailablePermissions.internal_webhooks_write,
    ]
```
The rest of the permissions are project-level

File: website/docs/docs/cloud/manage-access/enterprise-permissions.md [hunk 1]
```diff
@@ -19,6 +20,8 @@ The following permission sets are available for assignment in dbt Cloud Enterpri
 can be granted to dbt Cloud groups which are then in turn granted to users. A dbt Cloud group
 can be associated with more than one permission sets.
 
+<Snippet src="enterprise-permissions-table" />
```
Comment (nghi-ly) [inline website/docs/docs/cloud/manage-access/enterprise-permissions.md]:
```suggestion
<Snippet path="enterprise-permissions-table" />
```

File: website/docs/docs/cloud/manage-access/enterprise-permissions.md [hunk 2]
```diff
@@ -13,200 +15,11 @@ help manage access controls within a dbt Cloud account. See the docs on [access
 control](/docs/cloud/manage-access/about-user-access) for more information on Role-Based access
 control (RBAC).
 
-## Permission Sets
+## Roles and permissions
```
Comment (runleonarun) [inline website/docs/docs/cloud/manage-access/enterprise-permissions.md:18]: Much better name!

File: website/docs/docs/cloud/manage-access/enterprise-permissions.md [hunk 3]
```diff
@@ -13,200 +15,11 @@ help manage access controls within a dbt Cloud account. See the docs on [access
...
-
-### Stakeholder
-- **Has permissions on:** Authorized projects
-- **License restrictions:** Intended for use with Read-Only licenses, but may be used with Developer licenses.
-
-Stakeholders can perform the following actions in projects they are assigned to:
-- View generated documentation
-- View generated source freshness reports
-- View the Read-Only dashboard
+<Permissions feature={'/snippets/_enterprise-permissions-table.md'} />
```
Comment (runleonarun) [inline website/docs/docs/cloud/manage-access/enterprise-permissions.md:22]: Love that it's a reusable 🥇

File: website/snippets/_enterprise-permissions-table.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+| Profile                 |       W       |               |       W         |                |   R    |
+| Projects                |       W       |               |       W         |       R        |   R    |
+| Repositories            |       W       |               |       W         |                |   R    |
+| Runs                    |       W       |               |       W         |                |   R    |
+| Semantic Layer Config   |    W          |               |       W         |                |   R    |
+
+
+### Project role permissions
+
+The project roles are intended for users who will be working within the projects in various capacities. They'll primarily have access to project-level permissions such as repos and the IDE, but may also have some account-level permisions.
```
Comment (runleonarun) [inline website/snippets/_enterprise-permissions-table.md]:
```suggestion
The project roles enable you to work within the projects in various capacities. They primarily provide access to project-level permissions such as repos and the IDE, but may also provide some account-level permissions.
```

File: website/snippets/_enterprise-permissions-table.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+* (W)rite &mdash; Create new or modify existing. Includes `send`, `create`, `delete`, `allocate`, `modify`, and `read`.
+* (R)ead &mdash; Can view but can not create or change any fields.
+
+Permissions: 
+
+* Account-level permissions &mdash; Permissions related to management of the dbt Cloud account. For example, billing and account settings.
+* Project-level permissions &mdash; Permissions related to the projects in dbt Cloud. For example, repos and access to the IDE. 
+
+### Account roles
+Account roles are primarily designed for managing the dbt Cloud account. Permissions to alter the account settings (for example, generating service tokens, inviting users, configuring SSO). They may also have access to project-level permissions as well. The **Account Admin** role is the highest level of access that can be assigned.
```
Comment (runleonarun) [inline website/snippets/_enterprise-permissions-table.md]: I suggest we address the user directly so we can make these sentences active.
```suggestion
Account roles enable you to manage the dbt Cloud account and manage the account settings (for example, generating service tokens, inviting users, configuring SSO). They also provide project-level permissions. The **Account Admin** role is the highest level of access you can assign.  
```

**Review by runleonarun — COMMENTED** [review]
@matthewshaver Just some wording requests!

**Review by runleonarun — APPROVED** [review]
🚢 it!

**Pull request merged**